### ISSUE TITLE
Sidenav collapse menu item cannot be focused

### ISSUE THREAD

**Issue by gnomeontherun** [vmware/clarity#3526]
## Describe the bug

The sidenav collapsable nav menu titles are not focusable with keyboard.

## How to reproduce

https://clarity.design/documentation/sidenav

- Using the keyboard alone, try to tab to the  'Collapsible Nav' control and observe that you cannot

## Expected behavior

It should be focusable.

## Versions

**App**

* Angular: [e.g. 6]
* Node: [e.g. 8.10.0]
* Clarity: [e.g. 0.12.5]

**Device:**

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

## Additional notes

Use a button for this control, also use aria-expanded and aria-controls like an accordion, or simply use an accordion.

**Comment by gnomeontherun** [comment]
This should be resolved.

**Comment by gnomeontherun** [comment]
Fixed by #4138

**Comment by chlane** [comment]
Collapsible Nav Element control now receives focus but cannot be activated using the keyboard. This control can only be used with a mouse . See. the Collapsible Nav Element example on https://clarity.design/documentation/sidenav.
